0.12/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:18.04
+
+ENV THRIFT_VERSION v0.12.0
+
+RUN buildDeps=" \
+		automake \
+		bison \
+		curl \
+		flex \
+		g++ \
+		libboost-dev \
+		libboost-filesystem-dev \
+		libboost-program-options-dev \
+		libboost-system-dev \
+		libboost-test-dev \
+		libevent-dev \
+		libssl-dev \
+		libtool \
+		make \
+		pkg-config \
+	"; \
+	apt-get update && apt-get install -y --no-install-recommends $buildDeps && rm -rf /var/lib/apt/lists/* \
+	&& curl -k -sSL "https://github.com/apache/thrift/archive/${THRIFT_VERSION}.tar.gz" -o thrift.tar.gz \
+	&& mkdir -p /usr/src/thrift \
+	&& tar zxf thrift.tar.gz -C /usr/src/thrift --strip-components=1 \
+	&& rm thrift.tar.gz \
+	&& cd /usr/src/thrift \
+	&& ./bootstrap.sh \
+	&& ./configure  --without-python --without-cpp \
+	&& make \
+	&& make install \
+	&& cd / \
+	&& rm -rf /usr/src/thrift \
+	&& curl -k -sSL "https://storage.googleapis.com/golang/go1.4.linux-amd64.tar.gz" -o go.tar.gz \
```

Comment:
Apache Thrift 0.12.0 supports go 1.7 or later, I believe... what is go 1.4 doing in this repo if the goal is to provide a compiler?  All that's needed is C++ for that.

Reply:
This line is copied from `0.11/Dockerfile`.
There is no reason no more.